### ISSUE TITLE
[Chrome plugin] Tray icon + Notifications

### ISSUE THREAD

**Issue by bubuntux** [zhukov/webogram#90]
Would be nice to keep the "app" running on background and put a notification icon, similar to hangout application

**Comment by rumatoest** [comment]
Tray icon (like in hangouts app) will be really  useful.

**Comment by AAraKKe** [comment]
Yes that would be a good idea. Please keep in mind the possibility of adding a feature like the one suggested in #664 to share anything right clicking on it, as can be done with pushbullet for instance. That would be really cool!
